cms/templates/js/course-outline.underscore

```diff
@@ -78,7 +78,7 @@ if (xblockInfo.get('graded')) {
                         </a>
                     </li>
                 <% } %>
-                <% if (xblockInfo.canBeDeleted()) { %>
+                <% if (xblockInfo.showSectionContent()) { %>
```

Comment:
@chrisndodge yes we have .canBeDeleted() in earlier PR. showSectionContent returning boolean.
If the xblock is an entrance exam then we hide some functionality like delete / drag.